src/components/Shop/Shop.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import "./Shop.css";
+export const Shop = () => {
+  return (
+    <div>
+      <main className="container">
+        <h1 className="container-title">store for bibliophiles</h1>
+
+        <button class="btn btn-link">
+          <a href="#" class="btn-text-decoration btn-shop ">
+            shop now
+          </a>
```

Comment:
Suggestion : You can use the useNavigate hook here instead of using anchor tag

Reply:
sure, I will use it